src/main/res/values/strings.xml

```diff
@@ -803,16 +803,19 @@
     <string name="autodel_server_ask">Do you want to delete %1$d messages now and all newly fetched messages \"%2$s\" in the future?\n\n⚠️ This includes e-mails, media and \"Saved messages\" in all server folders\n\n⚠️ Do not use this function if you want to keep data on the server\n\n⚠️ Do not use this function if you are using other e-mail clients besides Delta Chat</string>
     <!-- shown below enabled autodel_server-option, should be a summary of autodel_server_ask and remind about the impact -->
     <string name="autodel_server_enabled_hint">This includes e-mails, media and \"Saved messages\" in all server folders. Do not use this function if you want to keep data on the server or if you are using other e-mail clients besides Delta Chat.</string>
+    <string name="autodel_sever_warn_multi_device_title">Turn on at-once deletion</string>
+    <string name="autodel_sever_warn_multi_device">If you enable at-once deletion you cannot use multiple devices on this profile anymore.</string>
     <string name="autodel_confirm">I understand, delete all these messages</string>
     <!-- "At once" in the meaning of "Immediately", without any intervening time. -->
-    <string name="autodel_at_once">At once</string>
+    <string name="autodel_at_once">At once after Download</string>
```

Comment:
"Immediately after Download" sounds better also wrt translations

Reply:
iirc, we had lots of discussions wrt that wording that time, wnd ended in "at once"; i would keep that